### DISCUSSION TITLE
How to change database charset?

### DISCUSSION THREAD

**Discussion by vladyslav-mikhieiev** [prisma/prisma#6879] [Q&A] [answered]
I need to change the charset of the base. Does a prisma have such an opportunity?

**Comment by pr0ph0z** [comment, marked as answer]
Pretty sure that Prisma has no built-in function to change the database or table charset directly (please correct me if I wrong). But Prisma has a method to execute a raw query using `prisma.$executeRaw()` method and given that you can change your database charset using that method.

**Comment by shirshak55** [comment]
May be you can do that in migrations? I  am sure we cannot do it out of box atm.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
